google/resource_container_cluster.go

```diff
@@ -2298,6 +2303,10 @@ func expandClusterAddonsConfig(configured interface{}) *containerBeta.AddonsConf
 			Disabled:        addon["disabled"].(bool),
 			ForceSendFields: []string{"Disabled"},
 		}
+		if addon["load_balancer_type"] != "" {
+			// Currently we only allow setting load_balancer_type to INTERNAL
+			ac.CloudRunConfig.LoadBalancerType = "LOAD_BALANCER_TYPE_INTERNAL"
```

Comment:
It will be easier to add more values in the future if we pass through the `addon["load_balancer_type"]` value here and let the `ValidateFunc` handle validation.

Reply:
Done